microhapulator/cli/offtarget.py

```diff
@@ -0,0 +1,49 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2022, DHS.
+#
+# This file is part of MicroHapulator (https://github.com/bioforensics/microhapulator) and is
+# licensed under the BSD license: see LICENSE.txt.
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+
+import microhapulator.api as mhapi
+import sys
+
+
+def subparser(subparsers):
+    desc = "Calculate off target read mapping"
+    cli = subparsers.add_parser("offtarget", description=desc)
+    cli.add_argument("markerbam", help="alignment file of reads aligned to marker sequences")
+    cli.add_argument(
+        "refbam", help="alignment file in BAM format of reads aligned to complete reference genome"
```

Comment:
We'll probably want to be explicit here as well that we're expecting the coordinates to correspond to GRCh38.

Reply:
Edited this to specify hg38